test/unit/test_exports/test_derived_quantities/test_surface_flux.py

```diff
@@ -105,6 +107,7 @@ def test_compute_cylindrical(r0, radius, height, c_top, c_bottom):
         height (float): cylinder height
         c_top (float): concentration top
         c_bottom (float): concentration bottom
+        Soret (bool): the Soret flag
```

Comment:
```suggestion
        soret (bool): if True, Soret effect will be set
```

Reply:
I included your suggestions. Also, added this line to the docstrings of the spherical case.